examples/recipes/README.md

```diff
@@ -0,0 +1,30 @@
+# Recipes
```

Comment:
Is this file meant to be a duplicate of the file in the docs?

Reply:
It is, for the users who might be navigating there thru GitHub. According to GitHub Insights, `/examples` are pretty popular in our repo. I think we would be able to see if users actually do that via our internal analytics.